lib/cmd/fh3/keys/ssh/add.js

```diff
@@ -0,0 +1,52 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var fhreq = require("../../../../utils/request");
+var fs = require('fs');
+var fhc = require("../../../../fhc");
+
+module.exports = {
+  'desc' : i18n._('Add a SSH Key Management'),
```

Comment:
I tried to use the same name which the user can find into the studio. Do you think it will be better as "Add a new SSH key"?